core/src/main/java/tc/oc/pgm/kits/KitParser.java

```diff
@@ -435,6 +440,28 @@ private List<Color> parseColors(List<Node> nodes) throws InvalidXMLException {
     return colors;
   }
 
+  public ItemStack parseBanner(Element el) throws InvalidXMLException {
+    ItemStack itemStack = parseItem(el, Materials.BANNER);
+    BannerMeta meta = (BannerMeta) itemStack.getItemMeta();
+    DyeColor color = XMLUtils.parseDyeColor(XMLUtils.getRequiredAttribute(el, "base-color"));
+    List<org.bukkit.block.banner.Pattern> patterns = new ArrayList<>();
+    patterns.add(new org.bukkit.block.banner.Pattern(color, PatternType.BASE));
```

Comment:
This is incorrect, and needs to be version dependant
for modern, there's a different material for each base banner (eg: WHITE_BANNER)
for legacy, there's one material (BANNER) but the meta has a base color (BannerMeta#setBaseColor)

you can see an example of this separated behavior in the implementations of ColorUtils#createBanner.

You may be able to use the preexisting `ColorUtils#setColor(itemStack, color)` for a version-independent alternative, but you may need some testing to make sure it works in both legacy and modern, and maybe needs tweaks to ensure it works with banners if it doesn't work out of the box:
```suggestion
    ItemStack itemStack = parseItem(el, Materials.BANNER);
    DyeColor color = XMLUtils.parseDyeColor(XMLUtils.getRequiredAttribute(el, "base-color"));    
    COLOR_UTILS.setColor(itemStack, color);
```